### PR TITLE
Added Circle of Trust, Responsiveness of Menubar and Password Length Check

### PR DESCRIPTION
**Pull request by fatimarafiqui**
This includes the following:
- Circle of Trust (Excluding Twilio Integration)
- Responsiveness of Menubar for small screen resolutions
- Password length validation on client side.

circle

menu

@prathmeshranaut @MehaKaushik

### CODE REVIEW

File: includes/menu.php [hunk 1]
```diff
@@ -2,138 +2,138 @@
...
-    'text' => 'Logged in as: Username',
-    'url' => Router::getPageUrl(Router::HOME, Router::PAGE_ADDED_SOON)
+        'text' => 'Logged in as: Username',
+        'url' => Router::getPageUrl(Router::HOME, Router::PAGE_ADDED_SOON)
     );
     $menu['items'][] = array(
-    'text' => 'Logout',
-    'url' => Router::getPageUrl(Router::HOME, Router::PAGE_ADDED_SOON)
+        'text' => 'Logout',
+        'url' => Router::getPageUrl(Router::HOME, Router::PAGE_ADDED_SOON)
```
Comment (MehaKaushik) [inline includes/menu.php]: Are these 4 spaces or a single tab ?

File: javascripts/circle_of_trust.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+$(document).ready(function() {
+    $('button').attr("disabled", false);
+    //circle type - 1 whole, 0.5 half, 0.25 quarter
+    var type = 1,
+        //distance from center
+        radius = '200%',
+        //shift start from 0
+        start = -90,
+        $elements = $('li:not(:first-child)'),
+        //adj for even distro of elements when not full circle
```
Comment (MehaKaushik) [inline javascripts/circle_of_trust.js]: This comment is not very clear, please rewrite it. But do not make it longer.

File: includes/menu.php (hunk 1 above)
Comment (fatimarafiqui) [inline includes/menu.php]: 4 spaces. The file has been passed through the php code sniffer as well.

File: modules/User.php [hunk 3]
```diff
@@ -110,4 +115,55 @@ public function updateUserDetails($userData)
...
+            $comrades_str = implode(', ', $comrades);
+            $found_user_id = $this->getCircleOfTrust();
+
+            if ($found_user_id) {
+                $stmt = $DB_CONNECT->prepare("UPDATE `comrades` SET `comrade_details` = ? WHERE `user_id` = ?");
+                $stmt->bind_param('si', $comrades_str, $found_user_id);
+                $stmt->execute();
+                $result = $stmt->get_result();
+                $stmt->close();
+                var_dump('1');
```
Comment (prathmeshranaut) [inline modules/User.php]: Remove this data.

File: modules/User.php [hunk 4]
```diff
@@ -110,4 +115,55 @@ public function updateUserDetails($userData)
...
+            } else {
+                $stmt = $DB_CONNECT->prepare("INSERT INTO `comrades` (`user_id`, `comrade_details`) VALUES (?, ?)");
+                $stmt->bind_param('is', $this->user_id, $comrades_str);
+                $stmt->execute();
+                $result = $stmt->get_result();
+                $stmt->close();
+                var_dump('2');
```
Comment (prathmeshranaut) [inline modules/User.php]: Remove this dump.

**Comment by MehaKaushik** [comment]
Good work

**Pull request merged**